Fails to start with "ImageCacheSize = 0"

Navidrome fails to start with `ImageCacheSize = 0`. Discovered on 0.38.0 when trying to bisect #446.
Bisect yields ba8d2f5da869d18dcd1dda39844063e81523c844 is the first bad commit.

```
ba8d2f5da869d18dcd1dda39844063e81523c844 is the first bad commit
commit ba8d2f5da869d18dcd1dda39844063e81523c844
Author: Deluan <deluan@navidrome.org>
Date:   Fri Oct 30 00:14:37 2020 -0400

    Log when a cache has finished loading

 core/cache/file_caches.go | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)
```

Log:

```
 _   _             _     _                                                                                                                                                                                                                    
| \ | |           (_)   | |                                                                                                                                                                                                                   
|  \| | __ ___   ___  __| |_ __ ___  _ __ ___   ___                                                                                                                                                                                           
| . ` |/ _` \ \ / / |/ _` | '__/ _ \| '_ ` _ \ / _ \                                                                                                                                                                                          
| |\  | (_| |\ V /| | (_| | | | (_) | | | | | |  __/                                                                                                                                                                                          
\_| \_/\__,_| \_/ |_|\__,_|_|  \___/|_| |_| |_|\___|                                                                                                                                                                                          
                                        Version: dev                                                                                                                                                                                          
                                                                                                                                                                                                                                              
Loaded configuration from 'navidrome.toml': &conf.configOptions{  
    ConfigFile:                 "navidrome.toml",                                                                                                                                                                                    [82/1998]
    Address:                    "0.0.0.0",                                                                                                                                                                                                    
    Port:                       4533,                                                                                                                                                                                                         
    MusicFolder:                "/storage/SyncRoot/Music/music",                                                                                                                                                                              
    DataFolder:                 "/var/lib/navidrome",                                                                                                                                                                                         
    DbPath:                     "/var/lib/navidrome/navidrome.db?cache=shared&_busy_timeout=15000&_journal_mode=WAL&_foreign_keys=on",                                                                                                        
    LogLevel:                   "debug",                                                                                                                                                                                                      
    ScanInterval:               60000000000,                                                                                                                                                                                                  
    SessionTimeout:             86400000000000,                                                                                                                                                                                               
    BaseURL:                    "/music",                                                                                                                                                                                                     
    UILoginBackgroundURL:       "https://source.unsplash.com/random/1600x900?music",                                                                                                                                                          
    EnableTranscodingConfig:    false,                                                                                                                                                                                                        
    TranscodingCacheSize:       "100MB",                                                                                                                                                                                                      
    ImageCacheSize:             "0",                                                                                                                                                                                                          
    AutoImportPlaylists:        true,                                                                                                                                                                                                         
    SearchFullString:           false,                                                                                                                                                                                                        
    IgnoredArticles:            "The El La Los Las Le Les Os As O A",                                                                                                                                                                         
    IndexGroups:                "A B C D E F G H I J K L M N O P Q R S T U V W X-Z(XYZ) [Unknown]([)",                                                                                                                                        
    ProbeCommand:               "ffmpeg %s -f ffmetadata",                                                                                                                                                                                    
    CoverArtPriority:           "embedded, cover.*, folder.*, front.*",                                                                                                                                                                       
    CoverJpegQuality:           75,                                                                                                                                                                                                           
    UIWelcomeMessage:           "",                                                                                                                                                                                                           
    GATrackingID:               "",                                                                                                                                                                                                           
    AuthRequestLimit:           5,                                                                                                                                                                                                            
    AuthWindowLength:           20000000000,                                                                                                                                                                                                  
    Scanner:                    conf.scannerOptions{Extractor:"ffmpeg"},                                                                                                                                                                      
    LastFM:                     conf.lastfmOptions{ApiKey:"", Secret:"", Language:"en"},                                                                                                                                                      
    Spotify:                    conf.spotifyOptions{},                                                                                                                                                                                        
    DevLogSourceLine:           false,                                                                                                                                                                                                        
    DevAutoCreateAdminPassword: "",                                                                                                                                                                                                           
    DevPreCacheAlbumArtwork:    false,                                                                                                                                                                                                        
    DevDisableTrackCoverArt:    false,                                                                                                                                                                                                        
    DevNewCacheLayout:          false,                                                                                                                                                                                                        
}                                                                                                                                                                                                                                             
DEBU[0000] Opening DataBase                              dbPath="/var/lib/navidrome/navidrome.db?cache=shared&_busy_timeout=15000&_journal_mode=WAL&_foreign_keys=on" driver=sqlite3                                                          
2020/11/22 19:24:57 goose: no migrations to run. current version: 20201110205344                                                                                                                                                              
INFO[0000] Starting scanner                              interval=1m0s                                                                                                                                                                        
INFO[0000] Configuring Media Folder                      name="Music Library" path=/storage/SyncRoot/Music/music                                                                                                                              
WARN[0000] Image cache disabled                                                                                                                                                                                                               
INFO[0000] Finished initializing cache                   cache=Image elapsedTime="400.222µs" maxSize=0                                                                                                                                        
panic: runtime error: index out of range [3] with length 3                                                                                                                                                                                    
                                                           
goroutine 6 [running]:                      
github.com/deluan/navidrome/log.addFields(0xc00038dd50, 0xc000267f18, 0x3, 0x3, 0xc0002c5e50)
        /var/lib/navidrome/x/navidrome/log/log.go:171 +0x397                            
github.com/deluan/navidrome/log.parseArgs(0xc000267f08, 0x4, 0x4, 0xc000267e60, 0x414fbb, 0x10)
        /var/lib/navidrome/x/navidrome/log/log.go:142 +0x433                                                                                                                                                                                  
github.com/deluan/navidrome/log.Debug(0xc000267f08, 0x4, 0x4)                                                          
        /var/lib/navidrome/x/navidrome/log/log.go:114 +0x56                                                            
github.com/deluan/navidrome/core/cache.NewFileCache.func1(0xc00007eba0)                                                
        /var/lib/navidrome/x/navidrome/core/cache/file_caches.go:51 +0x3a5
created by github.com/deluan/navidrome/core/cache.NewFileCache                                                         
        /var/lib/navidrome/x/navidrome/core/cache/file_caches.go:39 +0xf9    
```

--- a/core/cache/file_caches.go
+++ b/core/cache/file_caches.go
@@ -49,7 +49,7 @@ func NewFileCache(name, cacheSize, cacheFolder string, maxItems int, getReader R
 		log.Info("Finished initializing cache", "cache", fc.name, "maxSize", fc.cacheSize, "elapsedTime", time.Since(start))
 		fc.ready = true
 		if fc.disabled {
-			log.Debug("Cache DISABLED", "cache", fc.name, "size")
+			log.Debug("Cache DISABLED", "cache", fc.name, "size", fc.cacheSize)
 		}
 	}()
 

